app/root.tsx

```diff
@@ -187,6 +188,23 @@ function Document({ children }: { children?: React.ReactNode }) {
           Skip to main content
         </a>
         <Progress />
+        <AnnounceBanner message="New GCN Circulars service starts April 17, 2023.  Learn more at one of these introductory webinars.">
```

Comment:
Remove the extra whitespace. Whitespace is collapsed in HTML.

```suggestion
        <AnnounceBanner message="New GCN Circulars service starts April 17, 2023. Learn more at one of these introductory webinars.">
```

Also, add a link to the news item.